x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_alerts/alert_status.cy.ts

```diff
@@ -39,6 +42,7 @@ import { ALERTS_URL } from '../../../urls/navigation';
 
 // FLAKY: https://github.com/elastic/kibana/issues/169091
```

Comment:
I have a PR trying to fix the flake, just haven't gotten back to it yet! https://github.com/elastic/kibana/pull/171676